allennlp/data/token_indexers/token_indexer.py

```diff
@@ -16,9 +17,22 @@ class TokenIndexer(Generic[TokenType], Registrable):
     34), or as lists of character IDs (e.g., "cat" gets represented by the numbers [23, 10, 18]),
     or in some other way that you can come up with (e.g., if you have some structured input you
     want to represent in a special way in your data arrays, you can do that here).
+
+    Parameters
+    ----------
+    token_min_padding_length : ``int``, optional (default=``0``)
+        The minimum padding length required for the :class:`TokenIndexer`. For example,
+        the minimum padding length of :class:``SingleIdTokenIndexer`` is the largest size of
```

Comment:
Sphinx is complaining about how you've formatted stuff here - you have two backticks after the `:class:` directive, instead of one (the one on the line above is ok, the ones on this line and below are not).